plugin.video.hak5/addon.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+# -*- coding: UTF-8 -*-
+
+#
+# Imports
+#
+import os
+import sys
+import urlparse
+import xbmc
+import xbmcaddon
+
+reload(sys)
+sys.setdefaultencoding('utf8')
+
+LIB_DIR = xbmc.translatePath(
+    os.path.join(xbmcaddon.Addon(id="plugin.video.hak5").getAddonInfo('path'), 'resources', 'lib'))
```

Comment:
any reason for you just not to use relative imports? 

`from resources.lib import something`?